packages/typespec-test/test/widget_dpg/generated/typespec-ts/src/api/operations.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import "../models/index.js";
```

Comment:
will need to remove this. or replace the the import from individual file under models folder.